src/app/app.component.html

```diff
@@ -1,9 +1,39 @@
-<dx-toolbar id="toolbar" style="margin-bottom: 21px">
-  <dxi-item
-    widget="dxButton"
-    [options]="buttonOptions"
-    location="after"
-  ></dxi-item>
-</dx-toolbar>
+<div
+  class="main-container"
+  [ngStyle]="
+    router.url === '/table'
+      ? {
+          'background-color': '#fff',
+          position: 'fixed',
+          top: '0',
+          left: '0',
+          width: '100vw',
+          height: '100vh'
+        }
+      : {
+          'background-color': '#fff'
+        }
+  "
+>
+  <div
```

Comment:
if this part only appears on the table component would be better to move it in the specific component